src/components/Breadcrumb/Breadcrumb.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
```

Comment:
also if i can add those two screenshots that are not the same as the design provided

![Screenshot 2020-06-15 at 1 29 13 PM](https://user-images.githubusercontent.com/6433679/84647699-73790200-af0c-11ea-99b5-ceb181fdc267.png)

design provided
![image](https://user-images.githubusercontent.com/6433679/84647724-7bd13d00-af0c-11ea-8cf0-fbebfa7c4d2a.png)

![Screenshot 2020-06-15 at 1 29 10 PM](https://user-images.githubusercontent.com/6433679/84647703-74119880-af0c-11ea-8412-d3857f63033e.png)

Reply:
I will fix the margins, the width, the text alignment and the shadow. Are there any colour codes for the collapsed view and the hovered areas? Should we use a svg instead of text (3 dots) for the collapsed items ?